package.json

```diff
@@ -16,7 +16,7 @@
     "test": "npm run lint && npm run test-only",
     "test-only": "mocha -c --require test/setup.js --recursive ./test",
     "test-app": "./bin/nodejs-dashboard.js node test/app/index.js",
```

Comment:
This one is weird.  As is, with the `./`, I get the following error:

> ./bin/nodejs-dashboard.js node test/app/index.js

'.' is not recognized as an internal or external command,
operable program or batch file.

When I change it to not have the `./` I get this error instead:

> nodejs-dashboard@0.4.1 test-app C:\Users\formidable\Documents\nodejs-dashboard
> bin/nodejs-dashboard.js node test/app/index.js

'bin' is not recognized as an internal or external command,
operable program or batch file.